miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplay.kt

```diff
@@ -75,4 +89,17 @@ internal class MiniAppWebViewClient(private val loader: WebViewAssetLoader) : We
         view: WebView,
         request: WebResourceRequest
     ): WebResourceResponse? = loader.shouldInterceptRequest(request.url)
+
+    @Suppress("TooGenericExceptionCaught", "SwallowedException")
+    override fun onPageFinished(webView: WebView, url: String?) {
+        super.onPageFinished(webView, url)
+        try {
+            val inputStream = webView.context.assets.open("inject.js")
+            inputStream.bufferedReader().use(BufferedReader::readText)
+        } catch (e: Exception) {
+            null
+        }?.let {
+            webView.evaluateJavascript("javascript:(function (){$it})()") {}
```

Comment:
I think this doesn't need the `javascript:` prefix or the immediately invoked function. It could be just `webView.evaluateJavascript(it) {}`.